WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -65,9 +65,8 @@ class LoginTests: XCTestCase {
 
     // Unified WordPress.com login/out
     // Replaces testWpcomUsernamePasswordLogin
-    func testWpcomLogin() {
-        _ = PrologueScreen().selectSiteAddress()
-            .proceedWithWP(siteUrl: "https://wordpress.com")
+    func testWPcomLogin() {
```

Comment:
>  and needed a capital 'p', dangit!

🤣 So true! As soon as I saw the name I thought the same. Well done 👏 

By the same token, what do you think of 

```suggestion
    func testWPComLogin() {
```

Reply:
Looking through the WPiOS code, we have literally no consistency on this. It's WPcom, WPCom, and even wpCom, wpcom or WPCOM in various places.
WPcom feels *slightly* more "normal" to me since it lines up with WordPress.com, so I'm going to leave it that way, but if you have strong feelings I'm happy to change it 🤷🏼‍♀️